docs/semgrep-supply-chain/malicious-dependencies.md

```diff
@@ -0,0 +1,69 @@
+---
+slug: malicious-dependencies
+title: Detect and remove malicious dependencies
+hide_title: true
+description: Learn how Semgrep detects malicious dependencies and enable malicious dependency detection in your Supply Chain scans.
+tags:
+  - Semgrep Supply Chain
+---
+
+# Detect and remove malicious dependencies
+
+:::info
+This feature is in private beta. To join, reach out to [support](/support).
+:::
+
+**Malicious dependencies** are intentionally dangerous packages, designed to compromise systems, unlike vulnerable dependencies, which are accidentally risky. These threats include packages that have always been malicious, such as typo-squatting attacks, or packages that become malicious after an attacker compromises a maintainer or injects harmful code. In some of these cases, the maliciousness of a package affects a specific range of versions.
+
+Semgrep is able to detect malicious dependencies in your projects and in pull requests (PRs) or merge requests (MRs).
+
+This feature is enabled after opting in to the beta program.
+
+## Supported package managers and sources of information
```

Comment:
Note that there will be a follow-up PR overhauling the [Supported languages > SSC doc](https://semgrep.dev/docs/supported-languages) later down the line.